cpp/src/io/reader.cpp

```diff
@@ -247,21 +253,29 @@ las::Points Reader::GetPointsWithinBox(const Box &box, double min_resolution)
     return out;
 }
 
-int32_t Reader::GetDepthAtResolution(double resolution) const
+int32_t Reader::GetDepthAtResolution(double resolution)
```

Comment:
can this no longer be a const?

Reply:
Because I call Reader::GetAllChildren that's not const.